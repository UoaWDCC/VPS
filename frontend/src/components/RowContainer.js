import React from "react";
import styles from "../styling/RowContainer.module.scss";

export default function RowContainer({ children }) {
  return (
    <>
      <div className={styles.rowContainer}>{children}</div>
    </>
  );
}
