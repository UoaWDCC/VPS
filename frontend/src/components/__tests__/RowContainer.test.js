import React from "react";
import renderer from "react-test-renderer";
import { BrowserRouter } from "react-router-dom";
import RowContainer from "../RowContainer";

test("RowContainer component snapshot test", () => {
  const component = renderer.create(
    <BrowserRouter>
      <RowContainer>
        <div>test</div>
        <div>test</div>
      </RowContainer>
    </BrowserRouter>
  );

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
