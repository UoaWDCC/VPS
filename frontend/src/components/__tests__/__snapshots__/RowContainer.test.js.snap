// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RowContainer component snapshot test 1`] = `
<div
  className="rowContainer"
>
  <div>
    test
  </div>
  <div>
    test
  </div>
</div>
`;
